code/modules/mob/living/carbon/alien/humanoid/humanoid_defense.dm

```diff
@@ -1,3 +1,18 @@
+/mob/living/carbon/alien/humanoid/hitby(atom/movable/AM, skipcatch = 0, hitpush = 0, blocked = 0)
+	hitpush = 0
+	var/obj/item/I
+	if(istype(AM, /obj/item))
```

Comment:
This should be an `!istype()` check with an inner body of `return ..()`

Reply:
Doing so will break things, as some of this behavior is dependent on calls up the path.

Removing the notification though, as that is spamming, and can conflict with later notices.